docs/integrations/android-specific-workflow-steps/gradle-runner.md

```diff
@@ -0,0 +1,48 @@
+
+### Prerequisites
+
+The workflow steps that need to be executed before running the **Gradle Runner** workflow step, along with their respective reasons, are listed in the table below.
+
+| Prerequisite Workflow Step                      | Description                                     |
+|-------------------------------------------------|-------------------------------------------------|
+| [**Git Clone**](https://docs.appcircle.io/workflows/common-workflow-steps/#git-clone) | To initiate the **Gradle Runner** process, the repository that needs to be built must be fetched from the branch. This is achieved as follows: Upon completion of the **Git Clone** step, it generates the `AC_REPOSITORY_DIR` variable, which is then used as the input for the **Gradle Runner** step. |
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/android-workflow-components-gradle-runner_1.png'/>
+
+### Input Variables
+For each component, specific input variables are required for its operation on your system. The input variables necessary for the **Gradle Runner** are as follows:
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/android-workflow-components-gradle-runner_2.png' />
+
+| Variable Name                 | Description                                    | Status |
+|-------------------------------|------------------------------------------------|--------|
+| `$AC_REPOSITORY_DIR`         | This variable represents the path of the cloned Git repository. If this step runs after the [Git Clone](https://docs.appcircle.io/workflows/common-workflow-steps/#git-clone) step, the variable will be automatically populated. | Required |
+| `$AC_MODULE`                 | This variable specifies the project module to be built. This variable can also be set via the build [Configuration](https://docs.appcircle.io/build/build-profile-configuration/). In Android Studio, you can locate the available modules for your project. For more information, please refer to this [Android document](https://developer.android.com/studio/projects#ApplicationModules). | Required |
+| `$AC_VARIANTS`               | This variable specifies the project variant to be built. This variable can also be set via the build [Configuration](https://docs.appcircle.io/build/build-profile-configuration/). In Android Studio, you can find the available variants for your project. For more information, please refer to this [Android document](https://developer.android.com/build/build-variants). | Required |
+| `$AC_OUTPUT_DIR`               | This variable specifies the project variant to be built. This variable can also be set via the build [Configuration](https://docs.appcircle.io/build/build-profile-configuration/). In Android Studio, you can find the available variants for your project. For more information, please refer to this [Android document](https://developer.android.com/build/build-variants). | Required |
```

Comment:
@ftmzhrsckldr this is the same description with the AC_VARIANTS. Could you please fix the right description of AC_OUTPU_DIR

Reply:
@boztopuz Desc is fixed, can you review again?